requirements.txt

```diff
@@ -1 +1,2 @@
 discord.py>=2.1.0,<3.0.0
+aiohttp==3.8.3
```

Comment:
This should be fine with `aiohttp>=3.8,<4.0`? That way I don't have to bump package versions manually every now and then. :^)

Reply:
Makes sense :)